feat(bil): savings plans improvements MTA-5530 (#4431)
scaleway/docs-content: pages/billing/additional-content/understanding-savings-plans.mdx

    </Message>
2. Receive a discount on your monthly bill in exchange.
3. Have the discount applied to the total amount of resources you use, up to the committed amount.

    <Message type="important">
      Keep in mind that:
        - If you use less than what you committed to, you will still be billed for the committed amount with the Savings Plan discount applied.
        - If you use more resources than the committed amount, you will be charged the standard prices for the excess amount.
    </Message>

<Lightbox src="scaleway-savings-plan.webp" size="medium" alt="the image contains a graph showing a simulation of how the Savings Plan works for an account. X axis = month and Y axis = commitment amount in Euro. A horizontal line represents the committed amount. The graph displays the consumption of an account over three months. In the first month, the consumption is under the committed amount, so under the line. For the second month, the consumption is exactly the committed amount, so it joins the line. On the third and last month, the consumption went over the committed amount. Therefore, it goes over the line." />

### Example

> If you have Savings Plan with a committed amount of 900€ per month, a 10% discount is applied. You consumption within the Savings Plan will cost you 810€ per month.

Reviewer: ```suggestion
If you have a Compute Savings Plan with a committed amount of 900€ per month during 3 years, a 10% discount is applied. You consumption within the Savings Plan will cost you 810€ per month.
```

If in the month of March, you use:

| Number of Instances | Instance Type         | Hourly Price | Usage (in hours)  | Monthly total |
| ------------ | ------------------- | ------ | ------ |------ |
| 1        | DEV-1S |  0.008€   | 744  | 5.95€ |
| 1    | POP2-64C-256G      | 2.35€ | 381 | 895.35€ |
|   |  |      | **Total**  | 901.30€ |

Since the Savings Plan has a committed amount of 900€, you would be charged:

**810€** (the discounted monthly fee for the committed amount) **+** **1.03€** (the excess amount used above the committed amount, charged at the standard price) **=** **811.03€**.

> This means your total bill for March would be **811.03€**.

Reviewer: ```suggestion
This means your total untaxed bill for March would be **811.03€**.
```

When you purchase a Savings Plan, you:

1. Commit to a certain amount of spending per month for a set period of time (3 years, for example).
    <Message type="note">
      Commitment rates are calculated based on the public prices of resources before taxes and do not include any discounts.
    </Message>
2. Receive a discount on your monthly bill in exchange.
3. Have the discount applied to the total amount of resources you use, up to the committed amount.

    <Message type="important">
      Keep in mind that:

>         - If you use less than what you committed to, you will still be billed for the committed amount with the Savings Plan discount applied.

Reviewer 2: ```suggestion
        - If you use less than you committed to, you will still be billed for the committed amount with the Savings Plan discount applied.
```

        - If you use more resources than the committed amount, you will be charged the standard prices for the excess amount.
    </Message>

<Lightbox src="scaleway-savings-plan.webp" size="medium" alt="the image contains a graph showing a simulation of how the Savings Plan works for an account. X axis = month and Y axis = commitment amount in Euro. A horizontal line represents the committed amount. The graph displays the consumption of an account over three months. In the first month, the consumption is under the committed amount, so under the line. For the second month, the consumption is exactly the committed amount, so it joins the line. On the third and last month, the consumption went over the committed amount. Therefore, it goes over the line." />

### Example

> If you have a Compute Savings Plan with a committed amount of 900€ per month during 3 years, a 10% discount is applied. You consumption within the Savings Plan will cost you 810€ per month.

Reviewer 2: ```suggestion
If you have a Compute Savings Plan with a committed amount of 900€ per month over 3 years, a 10% discount is applied. Your consumption within the Savings Plan will cost you 810€ per month.
```
